falco/values.yaml

```diff
@@ -201,6 +201,8 @@ driver:
       # -- Extra environment variables that will be pass onto Falco driver loader init container.
       env: {}
       args: []
+      resources: {}
```

Comment:
Could you please add a comment for the new variables? just a short explanation of what they are.